BUILDING.md

```diff
@@ -26,7 +26,7 @@ Support is divided into three tiers:
   the broader community.
 * **Tier 2**: Full test coverage but more limited maintenance,
   often provided by the vendor of the platform.
-* **Experimental**: Known to compile but not necessarily reliably or with
+* **Experimental**: Known to compile but not necessarily reliable or with
```

Comment:
I changed it to `May not compile reliably or test suite may not pass.`. Is that okay?